dispatcher/backend/src/common/schemas/orms.py

```diff
@@ -0,0 +1,53 @@
+import marshmallow.fields as mf
+from marshmallow import post_dump
+from marshmallow_sqlalchemy import SQLAlchemySchema, auto_field
+
+import db.models as dbm
+from common.roles import get_role_for
+
+
+class BaseSchema(SQLAlchemySchema):
+    SKIP_VALUES = set([None])
+
+    @post_dump
+    def remove_skip_values(self, data, **kwargs):
+        if isinstance(data, (list, tuple, set)):
+            return type(data)(self.remove_skip_values(x) for x in data if x)
+        elif isinstance(data, dict):
+            return type(data)(
+                (k, self.remove_skip_values(v))
+                for k, v in data.items()
+                if self.remove_skip_values(v)
+            )
+        else:
+            return data
+
+
+class UserSchemaReadMany(BaseSchema):
```

Comment:
I confirm I've been very puzzled by all these stranges names.
I propose to postpone it to later, since it is not a blocker from my PoV + I would prefer to rework all names that are used in the end, not just the ones I'm touching.
I've added it to the checklist in main Issue.

Reply:
I finally spent a bit of time on this.
The `Schema` is indeed a marshmallow "thing". And it looks like "everyone" in the community is appending the `Schema` suffix to their classes, so I propose to decide to continue to use it.
Regarding the naming, `UserInListing` feels a bit too simple to me. I have no clue that this is meant to transform an object from the ORM (a list of `dbm.User` objects) into a JSON object meant to be used as a reply to the `GET /users` endpoint. I would suggest to be more specific and say `GetUsersRespSchema` : `RespSchema` to identify this is a Schema meant to be used to build an Http Response, `Get` to identify the HTTP Verb, and `Users` to identify the endpoint. An alternative could be `ListUsersRespSchema` to be more expressive about the action and less about the technical HTTP Verb.
WDYT ?

Reply:
OK to keep the `Schema` suffix if that's the convention. I figured the import location expressed that already.

I don't like how this is tied to the _HTTP GET_ response.
To me it's a special representation of some data (a User, as would appear in a listing) and it should be named based on what it represents, not how it's going to be used.
If it has a unique usage in a single endpoint, why are we not bundling it into the route? 

As you wrote, it's important for maintenance but it's not blocker for the migration. Use your own names and once it's over, we'll be able to see all at once and decide on a global renaming if necessary. Can you add it to the checklist?